ACORN/current_generator/acorn_constants.py

```diff
@@ -450,8 +449,8 @@
     [ "principal_investigator",       "Wyatt, Lucy" ],
     [ "citation",                     " The citation in a list of references is: IMOS, [year-of-data-download], [Title], [data-access-URL], accessed [date-of-access]" ],
     [ "acknowledgement",              "Data was sourced from the Integrated Marine Observing System (IMOS) - IMOS is supported by the Australian Government through the National Collaborative Research Infrastructure Strategy (NCRIS) and the Super Science Initiative (SSI)." ],
-    [ "distribution_statement",       "Data, products and services from IMOS are provided \"as is\" without any warranty as to fitness for a particular purpose" ],
-    [ "comment",                      "${extraComment}This NetCDF file has been created using the IMOS NetCDF User Manual v1.2. A copy of the document is available at http://imos.org.au/facility_manuals.html ." ]
+    [ "distribution_statement",       "Data may be re-used, provided that related metadata explaining the data has been reviewed by the user, and the data is appropriately acknowledged. Data, products and services from IMOS are provided \"as is\" without any warranty as to fitness for a particular purpose." ],
+    [ "comment",                      "${extraComment}This NetCDF file has been created using the IMOS NetCDF Conventions v1.4." ]
```

Comment:
This should be reflected in the `Conventions` attribute. It looks like you're still setting that to "CF-1.5,IMOS-1.2". In fact, if you update that, you don't really need this comment.